src/lib/utils.js

```diff
@@ -80,3 +83,13 @@ export function getFullLink(client, id = null) {
 export function getParentFolderLink(client, file) {
   return getFullLink(client, getParentFolderId(file))
 }
+
+export function getAppFullName() {
+  const dataset = getDataset()
```

Comment:
getDataSet() can be pretty extensive since we don't memoize it. You should use `client.getInstanceOptions()` and we should have a method for that https://github.com/cozy/cozy-notes/blob/master/src/components/app.jsx#L68-L69

Reply:
Well, if we cache it, it's not 2 ou 3 calls that will make a real difference

Reply:
Other argument; 

We plan to make getInstanceOption() works on mobile too (ie without having data injected by the stack). So it's better to rely on cozy-client for the futur, nop?